python3/relocate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script ensures that the installation's sysconfig module contains the
+# correct paths. Without this, any C modules will fail to compile.
+
+root="$1"
+cd "${root}"
+
+sysconfig="$(echo install/lib/python*/_sysconfigdata_*.py)"
+tmpfile="$(mktemp)"
+trap "rm -f ${tmpfile}" INT EXIT
+
+chmod +w "${sysconfig}"
+
+cp "${sysconfig}" "${tmpfile}"
+
+sed "s,'/install,'${root}/install,g" "${tmpfile}" > "${sysconfig}"
```

Comment:
```suggestion
sysconfig="$(echo ${root}/install/lib/python*/_sysconfigdata_*.py)"

chmod +w "${sysconfig}"

sed -i '' "s,'/install,'${root}/install,g" "${sysconfig}"
```

Why `tmpfile` instead of `sed -i ''`?

Reply:
Believe me, I tried - it doesn't work on OSX because it tries to create a backup in the same directory.